doctr/utils/geometry.py

```diff
@@ -145,22 +145,62 @@ def rotate_abs_boxes(boxes: np.ndarray, angle: float, img_shape: Tuple[int, int]
     return rotated_boxes
 
 
+def remap_boxes(boxes: np.ndarray, orig_shape: Tuple[int, int], dest_shape: Tuple[int, int]) -> np.ndarray:
+    """ Remaps a batch of RotatedBbox (x, y, w, h, alpha) expressed for an origin_shape to a destination_shape.
+    This does not impact the absolute shape of the boxes, but allow to calculate the new relative RotatedBbox
+    coordinates after a resizing of the image.
+
+    Args:
+        boxes: (N, 5) array of RELATIVE RotatedBbox (x, y, w, h, alpha)
+        orig_shape: shape of the origin image
+        dest_shape: shape of the destination image
+
+    Returns:
+        A batch of rotated boxes (N, 5): (x, y, w, h, alpha) expressed in the destination referencial
+
+    """
+
+    if len(dest_shape) != 2:
+        raise ValueError(f"Mask length should be 2, was found at: {len(dest_shape)}")
+    if len(orig_shape) != 2:
+        raise ValueError(f"Image_shape length should be 2, was found at: {len(orig_shape)}")
+    orig_height, orig_width = orig_shape
+    dest_height, dest_width = dest_shape
+    mboxes = boxes.copy()
+    mboxes[:, 0] = ((boxes[:, 0] * orig_width) + (dest_width - orig_width) / 2) / dest_width
+    mboxes[:, 1] = ((boxes[:, 1] * orig_height) + (dest_height - orig_height) / 2) / dest_height
+    mboxes[:, 2] = boxes[:, 2] * orig_width / dest_width
+    mboxes[:, 3] = boxes[:, 3] * orig_height / dest_height
```

Comment:
I will have to redo all the calculations to ensure, but aren't the remapped width & height depending on the alpha? Because they represent the width along the orientation (if the box has an alpha of 45°, the width is along the diagonal axis)

Reply:
Yes we discussed that, I tried to answer your concerns with a stricter testing in `test_remap_boxes` with a 45° inclined box. Width and Height are independent from alpha.